src/Integration/Binding/BindingWorkflow.cs

```diff
@@ -229,6 +229,9 @@ private ProgressStepDefinition[] CreateWorkflowSteps(IProgressController control
                 File.WriteAllText(tempRuleSetFilePath, export.Configuration.RuleSet.OuterXml);
                 RuleSet ruleSet = RuleSet.LoadFromFile(tempRuleSetFilePath);
 
+                // Remove/Move/Refactor me when we no longer download a XML file but actually use the proper API
```

Comment:
Let's change comment to: `Remove/Move/Refactor code when XML ruleset file is no longer downloaded but the proper API is used to retrieve rules`